src/app/auth/confirm-login/confirm-login.component.ts

```diff
@@ -58,19 +55,16 @@ export class ConfirmLoginComponent implements OnInit {
   completeLoginProcess(): void {
     this.authService.changeAuthState(AuthState.AwaitingAuthentication);
     this.phaseService.initializeCurrentRepository();
-    this.userService
-      .createUserModel(this.username)
-      .pipe(flatMap(() => this.githubEventService.setLatestChangeEvent()))
-      .subscribe(
-        () => {
-          this.handleAuthSuccess();
-        },
-        (error) => {
-          this.authService.changeAuthState(AuthState.NotAuthenticated);
-          this.errorHandlingService.handleError(error);
-          this.logger.info(`ConfirmLoginComponent: Completion of login process failed with an error: ${error}`);
-        }
-      );
+    this.userService.createUserModel(this.username).subscribe(
```

Comment:
Why is the `.pipe(flatMap(() => this.githubEventService.setLatestChangeEvent()))` line removed here?

Reply:
Apologies, I was experimenting with the method to understand what it produced and I must have forgot to reinsert it. I'm still not too sure what it does but I will reinstate the call.